### PR TITLE
Feat/sort memos by updated ts

### LINKED ISSUE #1261: Sort memos by updatedTs
### Is your feature request related to a problem?

I'm sure this is necessary for some users.

### Describe the solution you'd like

It is expected to select the sorting way directly on the home page.

There is no good implementation for now. I think it depends on the query API on the backend.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by titanventura**
Fixes #1261

### CODE REVIEW

File: web/src/components/Settings/PreferencesSection.tsx
Comment (CorrectRoadH) [inline web/src/components/Settings/PreferencesSection.tsx:53]:
```suggestion
  const handleSortMemosByUpdatedTsChanged = (event: React.ChangeEvent<HTMLInputElement>) => {
```

File: web/src/components/Settings/PreferencesSection.tsx
Comment (CorrectRoadH) [inline web/src/components/Settings/PreferencesSection.tsx:129]:
```suggestion
        <Switch className="ml-2" checked={localSetting.sortMemosByUpdatedTs} onChange={handleSortMemosByUpdatedTsChanged} />
```

**Review by CorrectRoadH — APPROVED** [review]
LGTM

File: web/src/components/Settings/PreferencesSection.tsx
Comment (titanventura) [reply web/src/components/Settings/PreferencesSection.tsx:53]: Ahh, Missed this. Thanks !

**Review by CorrectRoadH — APPROVED** [review]
LGTM

**Review by CorrectRoadH — APPROVED** [review]
LGTM. You can wait for the review from boojack. After it, the PR will be merged🥳

**Comment by titanventura** [comment]
@boojack Any updates on this ?

**Comment by Zeng1998** [comment]
I think we need to sort memos in the backend.

https://github.com/usememos/memos/blob/main/store/memo.go#LL287C24-L287C24

**Comment by boojack** [comment]
Implemented with https://github.com/usememos/memos/pull/1760, https://github.com/usememos/memos/pull/1761

**Pull request closed without merging**
